packages/blade/src/components/ProgressBar/_decisions/decisions.md

```diff
@@ -0,0 +1,102 @@
+# ProgressBar <!-- omit in toc -->
+A Progress bar is generally a branded element that indicates progress of process or task.
+
+<img src="./progress-bar-thumbnail.png" width="380" />
+
+- [Design](#design)
+- [API](#api)
+  - [Sample Usage](#sample-usage)
+  - [Accessibility](#accessibility)
+- [Handling pulsating animation](#handling-pulsating-animation)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+
+## Design
+- [Figma - ProgressBar](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=16430%3A256423)
+
+
+## API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| label | `string` | `undefined` | The label to be rendered with the progress bar. |  |
+| value | number | `undefined` | The current progress value for the progress bar. The progress bar would be in an indeterminate state if `progress` is not passed. |  |
+| contrast | `low`, `high` | `low` | The contrast of the progress bar to be rendered. |  |
+| intent | `positive`,`negative`, `notice`, `information`, `neutral` | `undefined` | The intent of the progress bar to be rendered. | |
+| size | `large`, `small` | `small` | The size of the progress bar to be rendered. |  |
+| accessibilityLabel | `string` | `"Loading"` | The accessibility label (aria-label) for the progress bar. |
+| isIndeterminate | `boolean` | `false` | Control whether the progress bar is indeterminate or not. The progress bar would be in an indeterminate state if `progress` is not passed.  |
```

Comment:
The issue with not having an explicit `isIndeterminate` prop is educating our consumers that such a feature exists without them checking the documentation. By adding an explicit prop we can reduce the education effort and provide the same behaviour when `value` is not passed

Reply:
Probably then we shouldn't set indeterminate behavior if `value` is `undefined` and keep that restricted to `isIndeterminate`. Otherwise we'll have two different ways to achieve the same results and possible combinations that may result in unexpected behavior.

Reply:
Hmm. Fair. We'll treat `undefined` as `0`